dev/MRTCore/packaging/native/Microsoft.ApplicationModel.Resources.props

```diff
@@ -5,5 +5,41 @@
 
   <Import Project="$(MSBuildThisFileDirectory)Microsoft.ApplicationModel.Resources.C.props" />
   <Import Project="$(MSBuildThisFileDirectory)Microsoft.ApplicationModel.Resources.WinRt.props" />
+  
+  <!-- <ItemGroup>
+    <AppxPackageRegistration Include="$(MSBuildThisFileDirectory)..\tools\AppX\x86\Release\Microsoft.ProjectReunion.0.1.appx">
+      <Architecture>x86</Architecture>
+      <Version>$(MicrosoftProjectReunionAppxVersion)</Version>
+      <Publisher>'CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US'</Publisher>
+    </AppxPackageRegistration>
+  </ItemGroup>
+  <ItemGroup>
+    <AppxPackageRegistration Include="$(MSBuildThisFileDirectory)..\tools\AppX\x86\Release\Microsoft.ProjectReunion.0.1.appx">
+      <Architecture>Win32</Architecture>
+      <Version>$(MicrosoftProjectReunionAppxVersion)</Version>
+      <Publisher>'CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US'</Publisher>
+    </AppxPackageRegistration>
+  </ItemGroup>
+  <ItemGroup>
+    <AppxPackageRegistration Include="$(MSBuildThisFileDirectory)..\tools\AppX\x64\Release\Microsoft.ProjectReunion.0.1.appx">
+      <Architecture>x64</Architecture>
+      <Version>$(MicrosoftProjectReunionAppxVersion)</Version>
+      <Publisher>'CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US'</Publisher>
+    </AppxPackageRegistration>
+  </ItemGroup>
+  <ItemGroup>
+    <AppxPackageRegistration Include="$(MSBuildThisFileDirectory)..\tools\AppX\arm\Release\Microsoft.ProjectReunion.0.1.appx">
+      <Architecture>arm</Architecture>
+      <Version>$(MicrosoftProjectReunionAppxVersion)</Version>
+      <Publisher>'CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US'</Publisher>
+    </AppxPackageRegistration>
+  </ItemGroup>
+  <ItemGroup>
+    <AppxPackageRegistration Include="$(MSBuildThisFileDirectory)..\tools\AppX\arm64\Release\Microsoft.ProjectReunion.0.1.appx">
+      <Architecture>arm64</Architecture>
+      <Version>$(MicrosoftProjectReunionAppxVersion)</Version>
+      <Publisher>'CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US'</Publisher>
+    </AppxPackageRegistration>
+  </ItemGroup> -->
```

Comment:
do we need this?

Reply:
There's an ongoing conversation about how to deal with this.  This AppxPackageRegistration is important for any VS project using this nuget, but the file path points to a local appx on the machine building the VS project, so we can't anticipate what that would be here.  So I left the commented out template here for when we eventually want to use it.

We may end up including the appx in the FWP, in which case we will uncomment this out.  But for now I think we should leave it in.

As an aside, this will really end up going into the ProjectReunion props file, not the MRTCore props file.  But that will come with a separate change once we actually start using the ProjectReunion props file.